client/static/how-to-pk6-2-homebrew.html

```diff
@@ -181,24 +180,21 @@ <h2 class="md-headline">Extracting the pk6 files</h2>
     <p>
 
     <ol>
-      <li>On the top directory of your SD card, there should be a new file named main, which you can save to your
-        computer. Once you have it on your computer, we recommend removing it from your SD card so you will not be
-        prompted to overwrite it the next time you perform this procedure.
-      </li>
       <li>Run KeySAV2. On the SAV tab, click on Open SAV and make sure that the file type dropdown in the bottom right is
         switched from SAV 1MB to Main file. Open your main file.
       </li>
-      <li>In the Options tab, select Select “To PK6 file” as the export style. On the SAV tab, choose the box number or
-        range you wish to view, and click Go to extract the PK6 files.
+      <li>In the Options tab, select Select "To PK6 file" as the export style. On the SAV tab, choose the box number or
+        range you wish to view, and click Go to extract the files.
```

Comment:
Is it worth waiting until #419 gets merged, and then telling people to just upload the save file itself?